### PR TITLE
[feature] matrix formatter: handle inf symbol

### PR DESCRIPTION
**Pull request by marehr**

### CODE REVIEW

File: include/seqan3/alignment/matrix/alignment_matrix_formatter.hpp [hunk 1]
```diff
@@ -52,6 +52,8 @@ struct alignment_matrix_format
     char const * row_sep{};
     //!\brief row separator symbol (a single symbol)
     char const * row_col_sep{};
+    //!\brief infinity symbol (a single symbol)
+    char const * inf{};
```
Comment (smehringer) [inline include/seqan3/alignment/matrix/alignment_matrix_formatter.hpp]: why don't you make it a char if it is a single symbol?

File: include/seqan3/alignment/matrix/alignment_matrix_formatter.hpp [hunk 2]
```diff
@@ -326,6 +328,8 @@ class alignment_matrix_formatter
         else
         {
             entry_type entry = matrix.at(row, col);
+            if(entry == matrix_inf<entry_type>)
```
Comment (smehringer) [inline include/seqan3/alignment/matrix/alignment_matrix_formatter.hpp]:
```suggestion
            if (entry == matrix_inf<entry_type>)
```
:nail_care:

File: include/seqan3/alignment/matrix/alignment_matrix_formatter.hpp [hunk 3]
```diff
@@ -136,31 +138,31 @@ struct alignment_matrix_format
...
 {
-    " ", ";", "", "",
+    " ", ";", "", "", "",
     {"N","D","U","DU","L","DL","UL","DUL"}
 };
 
 constexpr alignment_matrix_format alignment_matrix_format::ascii
 {
-    " ", "|", "-", "/",
+    " ", "|", "-", "/", "INF",
```
Comment (marehr) [inline include/seqan3/alignment/matrix/alignment_matrix_formatter.hpp]: this is more than a single char

File: include/seqan3/alignment/matrix/alignment_matrix_formatter.hpp [hunk 4]
```diff
@@ -136,31 +138,31 @@ struct alignment_matrix_format
...
     {" ","D","U","DU","L","DL","UL","DUL"}
 };
 
 constexpr alignment_matrix_format alignment_matrix_format::unicode_block
 {
-    "ε", "║", "═", "╬",
+    "ε", "║", "═", "╬", "∞",
```
Comment (marehr) [inline include/seqan3/alignment/matrix/alignment_matrix_formatter.hpp]: this is more than a single char

File: include/seqan3/alignment/matrix/alignment_matrix_formatter.hpp (hunk 1 above)
Comment (marehr) [reply include/seqan3/alignment/matrix/alignment_matrix_formatter.hpp]: because it is not a single symbol.

**Comment by marehr** [comment]
inf will now be tested

**Pull request merged**